ports/mimxrt10xx/common-hal/pwmio/PWMOut.c

```diff
@@ -240,7 +240,6 @@ pwmout_result_t common_hal_pwmio_pwmout_construct(pwmio_pwmout_obj_t *self,
 
         // Disable all fault inputs
         flexpwm->SM[submodule].DISMAP[0] = 0;
-        flexpwm->SM[submodule].DISMAP[1] = 0;
```

Comment:
is this deletion intentional?

Reply:
Ya, turns out that register doesn't exist. :-)